src/main/java/frc/trigon/robot/poseestimation/robotposesources/RobotPoseSource.java

```diff
@@ -76,11 +66,64 @@ public double getLastResultTimestamp() {
         return inputs.lastResultTimestamp;
     }
 
-    private Pose2d getUnCachedRobotPose() {
-        final Pose3d cameraPose = doubleArrayToPose3d(inputs.cameraPose);
+
+    /**
+     * Calculates the standard deviations of the estimated robot pose.
```

Comment:
What value does this javadoc add to the name of the function? 
Explain the logic that goes into this.